operator/config/samples/component-integration-installed/operator_v1alpha1_moduletemplate_cp-module.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: ModuleTemplate
+metadata:
+  name: moduletemplate-sample-cp-module
+  namespace: default
+  labels:
+    "operator.kyma-project.io/managed-by": "kyma-operator"
+    "operator.kyma-project.io/controller-name": "cp-operator"
+    "operator.kyma-project.io/module-name": "cp-module"
+    "operator.kyma-project.io/profile": "production"
+  annotations:
+    "operator.kyma-project.io/module-version": "v0.0.37"
+    "operator.kyma-project.io/module-provider": "internal"
+    "operator.kyma-project.io/descriptor-schema-version": "v2"
+    "operator.kyma-project.io/generated-at": "2022-07-11T14:59:43Z"
+spec:
+  channel: stable
+  target: control-plane
+  data:
+    kind: ControlPlaneModule
+    resource: controlplanemodules
+    apiVersion: component.kyma-project.io/v1alpha1
+  descriptor:
+    component:
+      componentReferences: []
+      name: kyma-project.io/module/cp-module
+      provider: internal
+      repositoryContexts:
+        - baseUrl: ghcr.io/fake
+          componentNameMapping: urlPath
+          type: ociRegistry
+        - baseUrl: ghcr.io/fake/signed
+          componentNameMapping: urlPath
+          type: ociRegistry
+      resources:
+        - access:
+            digest: sha256:9d37fb366e4371b44c94496f3db276bd6f3a1019c9d44077b689c68ec6486c1e
+            type: localOciBlob
+          name: crds
+          relation: local
+          type: yaml
```

Comment:
crds are potentially many yamls, thus we should probably not use yaml since its not one file.

Reply:
what type do you suggest we define here, a compressed file? I'm thinking it can be an aggregated yaml include all, which also easy for deploy.

Reply:
Well in OCI usually you use a compressed directory in a VFS, cli already has an implementation for that for the charts since they are multiple files so we could simply reuse that here. Of course this still allows you to define a folder with only one huge aggregated yaml in if we want to go with your approach later on